### DISCUSSION TITLE
Hide Wizard Step on mobile

### DISCUSSION THREAD

**Discussion by mstfkhazaal** [filamentphp/filament#12676] [Help]
### Package

Actions

### Package Version

3.2.73

### How can we help you?

How can hide  steps wizard on mobile

![photo](https://github.com/filamentphp/filament/assets/47450090/f8557f86-ee5b-499d-bcce-d98ba04d8875)

**Comment by henninghorn** [comment]
Hi!

Add the following to your `app.css` file or similar:

```css
.fi-fo-wizard-header {
    @apply hidden sm:grid !important;
}
```

This ensures that the wizard header, where the steps are contained gets hidden by default, but turn into a grid when the `sm` breakpoint is reached: [https://tailwindcss.com/docs/responsive-design](https://tailwindcss.com/docs/responsive-design)

I hope this helps!

**Comment by ermix3** [comment]
Hello!

Add the following to your theme.css file, I think better

```
.fi-fo-wizard-header .fi-fo-wizard-header-step:not(.fi-active) {
        @apply hidden md:grid !important;
}
```

**Reply by FilipRastovic** [reply]
This should be a built in feature! My wizard with 10 steps makes no sense without this on mobile.
